meet-data/probe.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# vim: set ts=8 sts=4 et sw=4 tw=99:
+#
+# Library for probes for new meets
+
+from bs4 import BeautifulSoup
+import datetime
+import os
+import shutil
+import sys
+import urllib.request
+
+FEDDIR = os.path.dirname(os.path.realpath(__file__))
+
+def error(msg):
```

Comment:
This should probably be renamed `die`. I realize that I'm the one that named it!